gen/weighted_test.go

```diff
@@ -0,0 +1,41 @@
+package gen_test
+
+import (
+	"testing"
+
+	"github.com/leanovate/gopter/gen"
+)
+
+func TestWeighted(t *testing.T) {
+	weighted := gen.Weighted([]gen.WeightedGen{
+		{1, gen.Const("A")},
+		{2, gen.Const("B")},
```

Comment:
Maybe switch two lines here to demonstrate the need of the sort (or proof me wrong ;) )